src/styles/block/card.scss

```diff
@@ -0,0 +1,134 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
```

Comment:
better to use sass vars here instead of CSS

Reply:
not fixed